api/src/main/java/org/openmrs/module/fhir2/api/dao/FhirRelatedPersonDao.java

```diff
@@ -9,22 +9,8 @@
  */
 package org.openmrs.module.fhir2.api.dao;
 
-import javax.validation.constraints.NotNull;
-
-import java.util.Collection;
-
-import ca.uhn.fhir.rest.api.SortSpec;
-import ca.uhn.fhir.rest.param.DateRangeParam;
-import ca.uhn.fhir.rest.param.StringAndListParam;
-import ca.uhn.fhir.rest.param.TokenAndListParam;
 import org.openmrs.Relationship;
 
 public interface FhirRelatedPersonDao extends FhirDao<Relationship> {
```

Comment:
Is this class empty??

Reply:
Yes

Reply:
ok,If its empty then why could we need it

Reply:
We need it when we add any other functionality to the resource. You can refer to other dao interfaces for reference.